Makefile.am

```diff
@@ -20,7 +20,7 @@ SUBDIRS = \
 
 @DX_RULES@
 
-EXTRA_DIST = bootstrap CHANGELOG config/README
+EXTRA_DIST = bootstrap CHANGELOG config/README config/odp-linux.conf
```

Comment:
Should ODP install this header file somewhere?

Reply:
A good point. Would this be OK?

.../platform/Makefile.inc
```
configdir = $(sysconfdir)/odp
config_DATA = $(top_srcdir)/config/odp-$(with_platform).conf

```